code/modules/mob/living/simple_mob/subtypes/vore/solargrub.dm

```diff
@@ -109,8 +109,13 @@ var/global/list/moth_amount = 0 // Chompstation Addition, Rykka waz here. *pawst
 
 /mob/living/simple_mob/vore/solargrub/proc/death_star()
 	visible_message("<span class='warning'>\The [src]'s shell rips open and evolves!</span>")
+
+/*
+//Commenting this bit out. It's unncecessary, especially since we only use one form.
```

Comment:
If it's unneeded, then maybe remove it? No need for bloated code that's not even used.

Reply:
Just in case we ever get around to balancing lunar moths and I don't want to lose the method of doing that.

Reply:
Fair nuff